src/wrap.c

```diff
@@ -2074,12 +2120,12 @@ inspectLib(struct dl_phdr_info *info, size_t size, void *data)
 
         const char *exe_or_lib_name = info->dlpi_name[0] ? info->dlpi_name : g_proc.procname;
         char msg[256];
-        appview_snprintf(msg, sizeof(msg), "a modification to the GOT to use lib %s for function %s", file_from_maps_file, fname);
-        gotSecurity(fname, msg, exe_or_lib_name, file_from_maps_file);
+        appview_snprintf(msg, sizeof(msg), "a modification to the GOT to use lib %s for function %s", file_from_got_value, fname);
+        gotSecurity(fname, msg, exe_or_lib_name, file_from_got_value);
         notify(NOTIFY_LIBS, msg);
 
 next:
-        appview_free(file_from_maps_file);
+        appview_free(file_from_got_value);
```

Comment:
need to check for null

Reply:
it's safe to call appview_free() with a null pointer. from musl source:
void free(void *p)
{	if (!p) return;
....